src/flexalloc_xnvme_env.h

```diff
@@ -10,11 +10,12 @@
 #include <libxnvme.h>
 #include <libxnvme_lba.h>
 
-struct fla_sync_strp_params
+struct fla_strp_params
 {
   uint32_t strp_nobjs;
   uint32_t strp_nbytes;
   uint64_t obj_nlbs;
+  uint64_t strp_snbytes;
```

Comment:
Can we add some documentation as to what the new parameters is? While you are at it documentation for all the other members of the struct.

Reply:
Some of these are actually copies of what we have in the pool_entry. so the right thing to do here is to document the pool entry elements and not repeat them here but just pass a pointer to the pool element.

Reply:
scratch that. we do not have access to these in xnvme_env.c file. so I'll leave them as a separate struct. We can figure out how to put them together later.

Reply:
Done